src/react/components/CivicHeaderSection.tsx

```diff
@@ -1,34 +1,32 @@
 import { useAnchorWallet, useConnection } from "@solana/wallet-adapter-react";
-import React from "react";
-import { Badge, GatewayStatus, useGateway } from "@civic/solana-gateway-react";
+import { Badge, GatewayStatus, IdentityButton, useGateway } from "@civic/solana-gateway-react";
 import { PublicKey } from "@solana/web3.js";
 import { config } from "../../config";
-import { CivicButton } from "@components/buttons/CivicButton";
 
 interface Props {
 	gatekeeperNetwork: PublicKey;
-	className?: string;
 }
 
 export const CivicHeaderSection = (props: Props) => {
 	const wallet = useAnchorWallet();
 	const connection = useConnection();
-	const { gatewayStatus, gatewayToken } = useGateway();
+	const { gatewayStatus, requestGatewayToken } = useGateway();
 
 	return (
 		<>
 			{wallet?.publicKey &&
 			<>
-				{props.gatekeeperNetwork && <Badge
-					clusterName={config.clusterConfig.name}
-					gatekeeperNetwork={props.gatekeeperNetwork}
-					publicKey={wallet.publicKey}
-					connection={connection.connection}
-				/>}
-				{!gatewayToken && <CivicButton
-					className={props.className}
-					text={GatewayStatus[gatewayStatus]}
-				/>}
+				{props.gatekeeperNetwork && <div className={"navbar-button"}>
```

Comment:
last time we agreed to do little cleanups of the code and we agreed to use the 'header' terminology when talking about the buttons on top. maybe renaming this navbar-button class is a good little cleanup. stuff like that keeps the code healthy

Reply:
and use prettier. configure your editor